CHANGELOG.md

```diff
@@ -1,27 +1,24 @@
-<!-- latest_release 3.4.7 -->
-## [v3.4.7](https://github.com/inspec/train/tree/v3.4.7) (2021-01-11)
-
-#### Merged Pull Requests
-- Test on ruby 3.0, drop testing on ruby 2.4 [#657](https://github.com/inspec/train/pull/657) ([clintoncwolfe](https://github.com/clintoncwolfe))
+<!-- latest_release -->
```

Comment:
Changes to the CHANGELOG should happen automaticaly when the PR merges - including them on a PR will only introduce conflicts.  These changes should not be included.

Reply:
Definitely this got pushed by mistake will revert this one. Thanks @clintoncwolfe